src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -1055,7 +1071,14 @@ int fx_muxer_t::execute(const int argc, const pal::char_t* argv[])
     new_argv[1] = sdk_dotnet.c_str();
 
     trace::verbose(_X("Using dotnet SDK dll=[%s]"), sdk_dotnet.c_str());
-    return parse_args_and_execute(own_dir, own_dll, 1, new_argv.size(), new_argv.data(), false, host_mode_t::muxer, &is_an_app);
+    result = parse_args_and_execute(own_dir, own_dll, 1, new_argv.size(), new_argv.data(), false, host_mode_t::muxer, &is_an_app);
+
+    if (pal::strcasecmp(_X("--info"), argv[1]) == 0)
```

Comment:
SDK will get this info once https://github.com/dotnet/cli/issues/6255 is fixed, right? If so, given that the issue is created, we should make the right change here immediately. Between merge of this PR and the CLI issue being fixed, the window is extremely narrow for someone to need muxer info and we can point them to the issue if need be (and the muxer info details as well).